src/main/java/org/eclipse/basyx/extensions/aas/aggregator/mqtt/MqttAASAggregatorHelper.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (C) 2021 the Eclipse BaSyx Authors
+ * 
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ ******************************************************************************/
+package org.eclipse.basyx.extensions.aas.aggregator.mqtt;
+
+/**
+ * A helper class for MqttAASAggregator. 
```

Comment:
Please be a little bit more specific here, i.e., "A helper class containing string constants of topics used by the AASAggregator"

Reply:
Updated the class description.